Tools/machines/lxplus-cern/spack.yaml

```
@@ -0,0 +1,47 @@
+spack:
+# CUDA
+  definitions:
+  - cuda: []
+  - cuda: [cuda ^ncurses+symlinks, blaspp +cuda ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+# basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python ^ncurses+symlinks, py-pip ^ncurses+symlinks, py-mpi4py ^ncurses+symlinks, py-numpy ^ncurses+symlinks,
+      py-periodictable ^ncurses+symlinks, py-picmistandard ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_PYTHON", "") == "1"
+  specs:
+  - ncurses+symlinks
+  - adios2 ^ncurses+symlinks
+  - blaspp ^ncurses+symlinks
+  - ccache ^ncurses+symlinks
+  - cmake ^ncurses+symlinks
+  - fftw ^ncurses+symlinks
+  - hdf5 ^ncurses+symlinks
+  - lapackpp ^ncurses+symlinks
+  - mpi ^ncurses+symlinks
+  - openpmd-api ^ncurses+symlinks
+  - pkgconfig
+  - $cuda
+  - $python
+
+  packages:
+    all:
+      target: [x86_64]
+      compiler: [gcc@9.2.0]
+  compilers:
+  - compiler:
+      environment: {}
+      extra_rpaths: []
+      spec: gcc@9.2.0
+      paths:
+        cc: /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0-afc57/x86_64-centos7/bin/gcc
+        cxx: /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0-afc57/x86_64-centos7/bin/g++
+        f77: /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0-afc57/x86_64-centos7/bin/gfortran
+        fc: /cvmfs/sft.cern.ch/lcg/releases/gcc/9.2.0-afc57/x86_64-centos7/bin/gfortran
+      flags: {}
+      operating_system: centos7
+      target: x86_64
+      modules: []
+  config:
+    build_jobs: 2
```

Same numbers as above: if `-j 6` is reasonable for the available head node resources, then do here:
```suggestion
    build_jobs: 6
```

---

Yes 6 seems reasonable